### PR TITLE
Modify documentation about NHC installation

### PR DESCRIPTION
**Pull request by omnivector-kelly**
Documentation for https://github.com/omnivector-solutions/slurm-charms/pull/151

### CODE REVIEW

File: operations/nhc.rst [hunk 1]
```diff
@@ -17,6 +17,20 @@ match your setup.
 When NHC identifies a node to be unhealthy, NHC drains this node to prevent
 future jobs from running on it.
 
+Acquire and Provide NHC
+==============
+Before the slurmd charm can run NHC must be installed, the .tar.gz must be supplied:
```
Comment (heitorPB) [inline operations/nhc.rst:22]:
```suggestion
Acquire and Provide NHC
=======================

Before the ``slurmd`` charm can run NHC must be installed, the ``.tar.gz`` must be supplied:
```

File: operations/nhc.rst [hunk 2]
```diff
@@ -17,6 +17,20 @@ match your setup.
...
+
+Acquire NHC
+------------
+  .. code-block:: bash
+    $ wget https://github.com/mej/nhc/releases/download/1.4.3/lbnl-nhc-1.4.3.tar.gz
```
Comment (heitorPB) [inline operations/nhc.rst:27]:
```suggestion
Acquire NHC
-----------

  .. code-block:: bash
  
    $ wget https://github.com/mej/nhc/releases/download/1.4.3/lbnl-nhc-1.4.3.tar.gz
```

File: operations/nhc.rst [hunk 3]
```diff
@@ -17,6 +17,20 @@ match your setup.
...
+
+Providing NHC
+------------
+  .. code-block:: bash
+    $ juju deploy slurmd --resource nhc=lbnl-nhc-1.4.3.tar.gz
```
Comment (heitorPB) [inline operations/nhc.rst:32]:
```suggestion
Providing NHC
-------------

  .. code-block:: bash

    $ juju deploy slurmd --resource nhc=lbnl-nhc-1.4.3.tar.gz
```

File: architecture.rst [hunk 4]
```diff
@@ -39,9 +39,10 @@ as part of the Slurm-core offering:
 * |slurmctld-badge|: Slurm control node (running ``slurmctld``)
 * |slurmrestd-badge|: Slurm REST service (running ``slurmrestd``)
 
-Additionally we include the `Node Health Check (NHC)
+Additionally we require the `Node Health Check (NHC)
```
Comment (heitorPB) [inline architecture.rst:42]: I did not get this, maybe it is a language issue? Is this understanding correct: "require" implies that the person adds NHC somehow to the system, while "include" means the charms do it, as part of the package?

File: new_words_spell_checker.txt [hunk 5]
```diff
@@ -57,3 +57,4 @@ Soltan
 systemd
 Tosta
 Zeestraten
+Heitor
```
Comment (heitorPB) [inline new_words_spell_checker.txt:60]: This is not needed anymore, Jaime's PR #75 adds it. Could you remove and rebase, please?

```suggestion
```

File: architecture.rst (hunk 4 above)
Comment (omnivector-kelly) [reply architecture.rst:42]: Right. It becomes a requirement for slurmd to operate vs. before where we would include the install of nhc.

**Comment by heitorPB** [comment]
This is how it looks right now: https://omnivector-solutions.github.io/osd-documentation/testing/operations/nhc.html

File: operations/nhc.rst [hunk 6]
```diff
@@ -17,6 +17,26 @@ match your setup.
...
+    
+     $ wget https://github.com/mej/nhc/releases/download/1.4.3/lbnl-nhc-1.4.3.tar.gz
+
+Providing NHC
+-------------
+
+.. code-block:: bash
+     
+     $ juju deploy slurmd --resource nhc=lbnl-nhc-1.4.3.tar.gz
+
```
Comment (heitorPB) [inline operations/nhc.rst:39]: This section should explain _when_ it is needed to do that. If getting the charms from Charmhub, this step is not needed, unless the person wants to change the version used. Thoughts?

Comment (omnivector-kelly) [reply operations/nhc.rst]: Please see [c20fbfa](https://github.com/omnivector-solutions/osd-documentation/pull/77/commits/c20fbfaef948698a0d8088fdeaf5dbdc2bd103d2) and let me know if that looks good.

**Review by heitorPB — COMMENTED** [review]
I think `Installation` instead of `Acquire and provide` is more intuitive, specially as it precedes the `configuration` section. What do you think?

**Pull request merged**